enclave-runtime/src/teeracle/mod.rs

```diff
@@ -139,6 +239,7 @@ where
 
 	let node_metadata_repository = GLOBAL_NODE_METADATA_REPOSITORY_COMPONENT.get()?;
 
+	// TODO: Ask Felix or Bigna how to get the extrinsic call index for my new extrinsic
 	let call_ids = node_metadata_repository
 		.get_from_metadata(|m| m.update_exchange_rate_call_indexes())
```

Comment:
You have already implemented:

```rust
fn update_oracle_call_indexes(&self) -> Result<[u8; 2]> {
		self.call_indexes(TEERACLE, "update_oracle")
	}
```

hence:

```rust
let call_ids = node_metadata_repository
		.get_from_metadata(|m| m.update_oracle_call_indexes())
                .map_err(Error::NodeMetadataProvider)?
```

Will give you the indexes of your new extrinsic.